server/modules/worker.js

```diff
@@ -0,0 +1,17 @@
+import client from '../helpers/redis';
+
+export default {
+  init: () => {
+    client.set('startTime', JSON.stringify(new Date()));
+  },
+  exec: () => {
+    // Set start time for the worker
+    client.get('jobDone', (err, value) => {
+      const count = Number(value) || 0;
```

Comment:
Since `Number('')` and `Number(null)` gives `0`, and `Number(undefined)` or `Number('random-string')` gives `NaN`, I think you may do away with the `||0` in line 10

Reply:
@nzediegwu1 if I do that, `count` will be set to `NaN` if a non integer value is passed. And that'll a bug

Reply:
Alright!